docs/content/developers/ontwerpkeuzes-informatiemodellen.md

```diff
@@ -0,0 +1,2 @@
+Bij het ontwikkelen van de API's aam de hand van user-story's kan voortschrijdend inzicht er toe leiden dat in de API cq. resource afgeweken wordt van het desbetreffende informatiemodel, i.c. RGBZ of ImZTC. Het desbetreffende informatiemodel moet daarop aangepast worden. Ook kan een (generieke) ontwerpkeuze (zie hier) als consequentie hebbe dat een informatiemodel aangpast moet worden.
```

Comment:
@ArjanKloosterboer heeft dit iets te maken met besluitregistratie? Het lijkt me dat dit een onafhankelijke pull-request had moeten zijn. #331 had volgens mij niet zomaar gemerged mogen worden.

Reply:
Ik weet eerlijk gezegd niet wat er nu gebeurd is. Ik wilde simpelweg een documentje toevoegen aan de map docs/content/developers. Maar Github is voor mij hier en daar nog steeds onnavolgbaar …
Kan jij iets herstellen of kan ik iets ontmergen?